packages/a/aui/xmake.lua

```diff
@@ -181,12 +288,37 @@ package("aui")
 
     on_test(function (package)
         assert(package:check_cxxsnippets({test = [[
+            #include <AUI/Platform/AWindow.h>
```

Comment:
the test package still lacks testing [aui assets](https://aui-framework.github.io/develop/aui-assets) functionality, what's the current state of it?

Reply:
https://aui-framework.github.io/develop/minimal-ui-template-with-assets